doc/api/single-executable-applications.md

```diff
@@ -0,0 +1,104 @@
+# Single executable applications
+
+<!--introduced_in=REPLACEME-->
+
+> Stability: 1 - Experimental: This feature is currently being designed and will
+> change.
+
+<!-- source_link=lib/internal/main/single_executable_application.js -->
+
+This feature allows the distribution of a Node.js application conveniently to a
+system that does not have Node.js installed.
+
+Node.js supports the creation of [single executable applications][] by allowing
+the injection of a JavaScript file into the `node` binary. During start up, the
+program checks if anything has been injected. If the script is found, it
+executes its contents, otherwise it operates like plain Node.js.
```

Comment:
The existing text is OK, but I would prefer one of these two alternatives:

```suggestion
program checks if anything has been injected. If the script is found, it
executes its contents.
```

```suggestion
program checks if anything has been injected. If the script is found, it
executes its contents. Otherwise it operates like plain Node.js.
```